.vscode/settings.json

```diff
@@ -8,5 +8,7 @@
     "**/node_modules": true
   },
   "typescript.surveys.enabled": false,
-  "editor.codeActionsOnSave": { "source.fixAll": true }
+  "editor.codeActionsOnSave": {
+    "source.fixAll": "explicit"
+  },
```

Comment:
I dont use vscode so I dont really care, but what does this do?

Reply:
Probably runs `eslint --fix` on save? 🤩 I use that myself (in WebStorm)

Reply:
Betyr at koden blir formattert når du lagrer filer eksplisitt. Egentlig ingen forskjell fra hva som var der, bare at `explicit` er nyere. 
Men tbh skulle jeg gjerne gitignora denne filen så man kan sette opp dette som man vil per repo. Feks liker jeg å sette farger på editoren sånn at jeg enkelt ser hvilket repo jeg holder på med, men det legges i denne fila. Og det har jeg ikke så lyst til å committe, for da får alle med vscode den fargen 😅